react_app/src/actions/actionTypes.js

```diff
@@ -1,6 +1,15 @@
+/* Authentication actions */
 export const CHECK_AUTHENTICATION_SUCCESS = "CHECK_AUTHENTICATION_SUCCESS";
 export const CHECK_AUTHENTICATION_FAILED = "CHECK_AUTHENTICATION_FAILED";
 export const LOGIN_SUCCESS = "LOGIN_SUCCESS";
 export const LOGIN_FAILED = "LOGIN_FAILED";
 export const LOGOUT = "LOGOUT";
 export const UPDATE_NEW_USER = "UPDATE_NEW_USER";
+
+/* Flag submission actions */
+export const SUBMIT_FLAG = "SUBMIT_FLAG";
+
+/* Game control actions */
+export const GET_GAME_STATE = "GET_GAME_STATE"
```

Comment:
convention here would be `FETCH_GAME_STATE`. You might also have `FETCH_GAME_STATE_SUCCESS` and FETCH_GAME_STATE_FAILURE`